[OSDOCS-9636]: ROSA docs is missing instructions related to the internal image backup (#71618)
openshift/openshift-docs: cloud_experts_tutorials/cloud-experts-deploy-api-data-protection.adoc

 name: ${CLUSTER_NAME}-dpa
 namespace: openshift-adp
spec:

>  backupImages:

Reviewer: Instead of keeping this value to empty, lets set it as true

Author: Done, thanks!

>  backupImages:true

Reviewer: ```suggestion
 backupImages: true
```

 name: ${CLUSTER_NAME}-dpa
 namespace: openshift-adp
spec:

>  backupImages:true

Reviewer: ```suggestion
 backupImages: true
```

[NOTE]
====

> * In OADP 1.1.x ROSA STS environments, the container image backup and restore (`spec.backupImages') value must be set to 'false' as it is not supported.

Reviewer 2: ```suggestion
* In OADP 1.1.x ROSA STS environments, the container image backup and restore (`spec.backupImages`) value must be set to `false` as it is not supported.
```

[NOTE]
====

> * Container image backup and restore (`spec.backupImages=false`) is disabled and not supported in OADP 1.1.x

Reviewer: Maybe we should re write this statement
```suggestion
In OADP 1.1.x set (`spec.backupImages=false`) as it's not supported.
```
